.github/workflows/BuildPR.yml

```diff
@@ -0,0 +1,25 @@
+name: PullRequest
+
+on: [pull_request]
+
+jobs:
+  buildGitx:
+    name: Build gitx
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.3
+        with:
+          fetch-depth: 0
+          submodules: true
+      - name: Set XCode Version
+        run: sudo xcode-select -s /Applications/Xcode_12.2.app
+      - name: Build project
+        run: xcodebuild -workspace MGScopeBar.xcodeproj/project.xcworkspace -scheme MGScopeBar archive | xcpretty
+      - name: Xcodebuild Action
+        uses: sersoft-gmbh/xcodebuild-action@v1.1
+        with:
+          project: MGScopeBar.xcodeproj
+          scheme: MGScopeBar
+          configuration: Release
+          action: archive
```

Comment:
Is `archive` or is `build` better ?

Reply:
Archive seems like a good idea, as it packages it up?

Reply:
I didn't check. Are there tests as well to run ?
Sorry, I don't now how to find it in Xcode. Maybe you tell it me